couchbase/example/src/test/java/example/springdata/couchbase/repository/AirlineRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,86 @@
+package example.springdata.couchbase.repository;
+
+import example.springdata.couchbase.config.CouchbaseConfiguration;
+import example.springdata.couchbase.model.Airline;
+import org.hamcrest.CoreMatchers;
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+import org.hamcrest.core.Is;
+import org.hamcrest.core.IsNull;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ *
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = CouchbaseConfiguration.class)
+public class AirlineRepositoryIntegrationTest {
+
+    @Autowired
+    private AirlineRepository airlineRepository;
+    private Airline a, b, c;
+    private List<Airline> airlineList;
+
+    @Before
+    public void setup() {
+        a = new Airline(20000,"airline", "CK 1", "LK", "CMB", "CLK","Sri Lanka");
+        b = new Airline(20001,"airline", "CK 2", "LK", "CMB", "CLK","Sri Lanka");
+        c = new Airline(20002,"airline", "CK 3", "LK", "CMB", "CLK","Sri Lanka");
+
+        airlineList = Arrays.asList(a, b, b);
+    }
+
+    @Test
+    public void testSaveAirline() {
+        Iterable<Airline> itr = airlineRepository.save(Arrays.asList(a));
+        Assert.assertThat(itr, Matchers.notNullValue());
```

Comment:
Please use AssertJ assertions and static import using a `*` so that static methods don't need to be qualified. See other projects for example.